tools/pre-commit

```diff
@@ -39,7 +39,23 @@ if (my @modified_files =
       push(@failed_files, $modified_file);    # Note the original file name
       $exit_code = $code; } }
 
-  print STDERR "Some files had issues: " . join(',', @failed_files)
-    . "; To correct these, run:\n tools/latexmllint <files>.\n"
-    . colored("COMMIT ABORTED", 'red') . "\n" if @failed_files;
+  if (@failed_files) {
+    print STDERR "Some files had issues: " . join(',', @failed_files)
+      . "; To correct these, run:\n tools/latexmllint <files>.\n"
+      . colored("COMMIT ABORTED", 'red') . "\n"; }
+  else {
```

Comment:
Probably better not to else here, but print any manifest messages in addition to any lint messages.